packages/graphback/src/InputTypeVisitor.ts

```diff
@@ -50,24 +50,42 @@ export const inputTypeVisitor = {
     }
   },
 
+  InterfaceTypeDefinition: (node: InterfaceTypeDefinitionNode) => {
+    let config = { create: false, update: false, subCreate: false, subUpdate: false, subDelete: false };
```

Comment:
[Documentation] So this implies that our logic is going to ignore anything that is applied to interface will be ignored and users should be applying those to types. 

This can be ensured by directive validation so it will not be possible to do it or we can simply mention that in docs.